src/Writers/HTMLWriter.jl

```diff
@@ -521,13 +524,14 @@ module RD
         ))
     end
     function mathengine!(r::RequireJS, engine::MathJax3)
+        url = isempty(engine.url) ? "https://cdnjs.cloudflare.com/ajax/libs/mathjax/3.0.5/es5/tex-svg.js" : engine.url
         push!(r, Snippet([], [],
             """
             window.MathJax = $(json_jsescape(engine.config, 2));
 
             (function () {
                 var script = document.createElement('script');
-                script.src = 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/3.0.5/es5/tex-svg.js';
+                script.src = $url;
```

Comment:
We need quotes here:

```suggestion
                script.src = '$url';
```

There is a `@test_broken` for this currently in the tests, so that would have to be updated as well.